### ISSUE TITLE
F-Script 2016 refresh

### ISSUE THREAD

**Issue by schrepfler** [Kentzo/F-Script#25]
With macOS, Swift3 and new XCode out, perhaps a F-Script refresher is in order?

**Comment by zmukwa** [comment]
Yes! In my opinion Swift should never have come about had F-Script caught on.Both are scripting languages but F-Script is superior and more compact and pure OOP. so yes please. Bring it on.

**Comment by Kentzo** [comment]
@schrepfler @zmukwa Anyone interested in pushing this branch forward should probably start with reviewing and merging https://github.com/Kentzo/F-Script/pull/20.
